### ISSUE TITLE
Kernel of Conv3D is larger than featuremap

### ISSUE THREAD

**Issue by 2053652473** [brjathu/deepcaps#22]
Thank you so much for your excellent work, I am very interested in your work. But I am a PYTORCH user, and it is a little difficult to read some content of TENSORFLOW. For DEEPCAPS28, I noticed that the feature size in CONV3D is 2*2, but KERNEL is 3*3, I would like to know the details of your PADDING. If possible, I hope you can tell me in natural language.
`    l_skip = ConvCapsuleLayer3D(kernel_size=3, num_capsule=32, num_atoms=8, strides=1, padding='same', routings=3)(l)`

**Comment by 2053652473** [comment]
I get it now
